HouseRules_Essentials/Rules/LevelSequenceOverriddenRule.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace HouseRules.Essentials.Rules
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Boardgame;
+    using Boardgame.LevelLoading;
+    using Boardgame.Networking;
+    using Data.GameData;
+    using DataKeys;
+    using HarmonyLib;
+    using HouseRules.Types;
+    using UnityEngine;
+
+    public sealed class LevelSequenceOverriddenRule : Rule, IConfigWritable<List<string>>, IMultiplayerSafe
+    {
+        public override string Description => "LevelSequence is overridden";
+
+        protected override SyncableTrigger ModifiedSyncables => SyncableTrigger.NewPieceModified | SyncableTrigger.StatusEffectDataModified;
```

Comment:
I'll try to get a friend to help me test this before merging. My expectation was that because the LevelSequence would be defined before anyone else joins the game, the state would be synced over when players join.

Reply:
Have now tested this successfully in multiplayer. No need to any special syncing.